scripts/merge_fusion/01-create-and-add-fusion-files.R

```diff
@@ -0,0 +1,124 @@
+# Author: Krutika Gaonkar
+# Merge step from Komal Rathi's code in OpenPBTA
+# Date: 05/10/2020
+# Function:
+# merges new RSEM files into previous version of rsem RDS object
+
```

Comment:
I'd update the Description  here which looks like I had just copied over from other scripts. 🤦‍♀️

```suggestion
# merges new arriba and starfusion files into previous released version of merged tsv files

```